src/it/scala/org/broadinstitute/dsde/rawls/WorkspaceSimulation.scala

```diff
@@ -0,0 +1,270 @@
+import java.util.logging.{Logger, LogManager}
+
+import akka.testkit.TestActorRef
+import com.orientechnologies.orient.client.remote.OServerAdmin
+import com.typesafe.config.ConfigFactory
+import org.broadinstitute.dsde.rawls.dataaccess._
+import org.broadinstitute.dsde.rawls.model._
+import org.broadinstitute.dsde.rawls.webservice.WorkspaceApiService
+import org.broadinstitute.dsde.rawls.workspace.WorkspaceService
+import org.broadinstitute.dsde.rawls.workspace.EntityUpdateOperations._
+import org.broadinstitute.dsde.rawls.WorkspaceSimulation._
+import org.scalatest.{Matchers, FlatSpec}
+import spray.http._
+import spray.json._
+import spray.httpx.SprayJsonSupport
+import SprayJsonSupport._
+import WorkspaceJsonSupport._
+import spray.testkit.ScalatestRouteTest
+import scala.concurrent.duration._
+
+object WorkspaceSimulation {
+  // TODO move these to config?
+  // TODO Rawls seems to become unresponsive if the numbers are much higher,
+  // TODO so we should try to gain a better understanding of the performance issues,
+  // TODO before trying to increase these numbers
+  val numSamples = 20
+  val numPairs = 10
+  val numSampleSets = 5
+  val numPairSets = 2
+  val numMethodConfigs = 20
+
+  val numAnnotationsSmall = 5
+  val numSetMembersSmall = 5
+
+  val numObjectUpdates = 10
+  val numAnnotationUpdatesSmall = 1
+
+  val numAnnotationsLarge = 1000
+  val numAnnotationUpdatesLarge = 200
+}
+
+class WorkspaceSimulation extends FlatSpec with WorkspaceApiService with ScalatestRouteTest with Matchers {
+  implicit val routeTestTimeout = RouteTestTimeout(600.seconds) // this is a load test, so response times may be slow
+  def actorRefFactory = system
+
+  // convenience methods - TODO add these to unit tests too?
+  def addCookie = addHeader(HttpHeaders.`Cookie`(HttpCookie("iPlanetDirectoryPro", "test_token")))
+  def httpJson[T](obj: T)(implicit writer: JsonWriter[T]) = HttpEntity(ContentTypes.`application/json`, obj.toJson.toString())
+  def repeat[T](n: Int)(exp: => T) = (1 to n) map (_ => exp)
+
+  // setup DB. if it already exists, drop and then re-create it.
+  val dbName = "integration-test-latest" // TODO move this into config?
+  val orientConfig = ConfigFactory.parseFile(new File("/etc/rawls.conf")).getConfig("orientdb")
+  val dbUrl = s"remote:${orientConfig.getString("server")}/${dbName}"
+  val admin = new OServerAdmin(dbUrl).connect(orientConfig.getString("rootUser"), orientConfig.getString("rootPassword"))
+  if (admin.existsDatabase()) admin.dropDatabase(dbName)
+  admin.createDatabase("graph", "plocal") // storage type is 'plocal' even though this is a remote server
+  val dataSource = DataSource(dbUrl, orientConfig.getString("rootUser"), orientConfig.getString("rootPassword"), 0, 30)
+
+  // suppress Java logging (otherwise OrientDB will produce a ton of useless log messages)
+  LogManager.getLogManager().reset()
+  Logger.getLogger(java.util.logging.Logger.GLOBAL_LOGGER_NAME).setLevel(java.util.logging.Level.SEVERE)
```

Comment:
I spent some time investigating it a while back, and I wasn't able to find a good solution since setting the ```java.util.logging``` level in config files or SBT doesn't seem to work. It might be worth looking into further, but not sinking too much time into it.